rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -813,6 +813,16 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 
 	lv2_sleep(ppu, 250);
 
+	// Check if RPCS3_BLIST section exist in PARAM.SFO
+	// This section contains the list of files in the save ordered as they would be in BSD filesystem
+	std::vector<std::string> blist;
+
+	if (psf.count("RPCS3_BLIST"))
+	{
+		std::string blist_content = psf.at("RPCS3_BLIST").as_string();
+		blist                     = fmt::split(blist_content, {"\n"}, false);
```

Comment:
Can be written in one line.
Also use delimiter `/`, because \n may be a valid in file names...